dvclive/live.py

```diff
@@ -30,23 +30,34 @@ def __init__(
         self,
         path: Optional[str] = None,
         resume: bool = False,
-        report: Optional[str] = "html",
+        report: Optional[str] = "auto",
     ):
         self._path: Optional[str] = path
         self._resume: bool = resume or env2bool(env.DVCLIVE_RESUME)
-        self._report: str = report
-        self.html_path = None
+
+        if report == "auto":
+            if env2bool("CI"):
+                report = "md"
+            else:
+                report = "html"
+        else:
+            if report not in {None, "html", "md"}:
+                raise ValueError(
+                    "`report` can only be `None`, `auto`, `html` or `md`"
+                )
+
+        self._report: Optional[str] = report
+        self.report_path = ""
 
         self.init_from_env()
 
         if self._path is None:
             self._path = self.DEFAULT_DIR
 
-        if self.html_path is None:
-            self.html_path = os.path.join(self.dir, "report.html")
-
         if self._report is not None:
-            out = Path(self.html_path).resolve()
+            if not self.report_path:
+                self.report_path = os.path.join(self.dir, f"report.{report}")
```

Comment:
@dberenbaum 

> Also, is there any way we can create some consistency between where html and md get written?

They are both written to `results/report.{format}`

Reply:
👍 I was probably looking at stale HTML reports.